style.css

```diff
@@ -0,0 +1,364 @@
+}
+
+.section-one-nav {
+  display: -ms-grid;
+  display: grid;
+  place-items: center;
+}
+
+.section-one-main__tweet-button:link, .section-one-main__tweet-button:visited {
+  font-size: 1rem;
+  padding: 0.4rem;
+}
+
+.section-one-nav__profile-image img,
+.section-one-main__profile-image img {
+  height: 3.5rem;
+  border-radius: 50%;
+}
+
+@media only screen and (max-width: 500px) {
+  .section-one-nav {
+    display: none;
+  }
+}
+
+.section-one-nav__list img {
+  height: 1.8rem;
+  padding-right: 0.6rem;
+}
+
+.section-one-nav__list,
+.section-one-nav li:first-child,
+.section-one-nav a {
+  padding-bottom: 1rem;
+}
+
+.section-one-navlist,
+.section-one-nav li:last-child,
+.section-one-nav a {
+  padding-top: 1rem;
+}
+
+.section-one-nav__list,
+.section-one-nav li,
+.section-one-nav a {
+  list-style: none;
+  margin: 0;
+  border: 0.1px solid transparent;
+  border-radius: 12px;
+  padding: 6px;
+}
+
+@media only screen and (max-width: 1282px) {
+  .section-one-nav__list,
+  .section-one-nav li,
+  .section-one-nav a {
+    color: transparent;
+    display: -ms-grid;
+    display: grid;
+    justify-items: center;
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.section-one-nav__list li a:link {
+  text-decoration: none;
+}
+
+.section-one-nav__list li:hover {
+  text-decoration: underline;
+  color: #636161;
+  -webkit-box-shadow: 0 5px 5px rgba(0, 0, 0, 0.15);
+          box-shadow: 0 5px 5px rgba(0, 0, 0, 0.15);
+  -webkit-transform: translateX(3px);
+          transform: translateX(3px);
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 1282px) {
+  .section-one-nav__list li:hover {
+    color: transparent;
+    display: -ms-grid;
+    display: grid;
+    justify-items: end;
+    border: 0;
+    -webkit-box-shadow: 0px 0 0px #636161;
+            box-shadow: 0px 0 0px #636161;
+    -webkit-transform: translateX(2px);
+            transform: translateX(2px);
+  }
+}
+
+.section-one-nav__list li:active {
+  -webkit-box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.15);
+          box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.15);
+  -webkit-transform: translateY(0);
+          transform: translateY(0);
+}
+
+.section-one-nav__list li:first-child:active,
+.section-one-nav li:first-child:hover,
+.section-one-nav li:last-child:active,
+.section-one-nav li:last-child:hover {
+  -webkit-box-shadow: 0 0 0 transparent;
+          box-shadow: 0 0 0 transparent;
+}
+
+.section-one-nav__list,
+.section-one-nav img {
+  position: relative;
+  top: 10px;
+}
+
+.section-one-nav__twitter-account-holder {
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-rows: 6rem auto;
+      grid-template-rows: 6rem auto;
+  margin: 0 2rem 0 0;
+  border-radius: 16px;
+  padding: 5px;
+}
+
+@media only screen and (max-width: 1282px) {
+  .section-one-nav__twitter-account-holder {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-rows: 5rem 6rem;
+        grid-template-rows: 5rem 6rem;
+    place-items: center;
+    margin: 0;
+  }
```

Comment:
Will the second `display: ` just overwrite the first one? 
And do you find easier to work with rem instead of pixels or is this generated by sass?

Reply:
1rem = 16px or your browsers default setting, so people with different default respond to rem. It's responsive.

The different display value is added by sass for other browsers